node/core/dispute-coordinator/src/real/ordering/tests.rs

```diff
@@ -197,7 +201,16 @@ async fn assert_block_ancestors_request(virtual_overseer: &mut VirtualOverseer,
 		AllMessages::ChainApi(ChainApiMessage::Ancestors { hash, k, response_channel }) => {
 			let maybe_block_position = chain.iter().position(|h| *h == hash);
 			let ancestors = maybe_block_position
-				.map(|idx| chain[..idx].iter().rev().take(k).copied().collect())
+				.map(|idx| {
+					if idx == 0 {
+						// Return zero hash for the genesis block -- the same way
+						// Chain API does.
+						vec![Hash::zero()]
```

Comment:
I am not convinced that this is sane. What would we ever do withthe zero hash? The genesis should not return ancestors, not a garbage one.

Reply:
I agree, although a check that we won't ever underflow at genesis is not a bad idea.